src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props

```diff
@@ -24,6 +24,15 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
   </PropertyGroup>
 
+  <!-- Default the value of PackRelease for every project. Note that this is pre-evaluated by the CLI in an evaluation before evaluation (see ReleasePropertyProjectLocator.cs).
+  Multitargeting pack splits a single 2 TFM project into 9 separate evaluations, with different target imports. Microsoft.NET.SDK.DefaultItems.Targets ...
+  is where PublishRelease is defaulted because it depends on _TargetFrameworkVersionWithoutV. Some targets do not run in some instances of dotnet pack.
+  So, we must set PackRelease here. Even though this occurs before project import, the project import can override PackRelease later, before the pre-evaluation, so this is fine.-->
+  <PropertyGroup Condition="'$(PackRelease)' == ''">
+    <!-- v Do not depend on this value being correct until after project evaluation.-->
+    <PackRelease>true</PackRelease>
```

Comment:
Same here, you probably need to set something like `_UsingDefaultPackRelease` to let the CLI know whether this was explicitly specified.

Reply:
As discussed offline, this occurs in the pre-evaluation, so it is not necessary.